packages/components/src/components/Avatar/index.tsx

```diff
@@ -5,6 +5,7 @@ import { Element } from '../Element';
 import { Text } from '../Text';
 
 interface IAvatarProps {
+  file?: string | null;
```

Comment:
**question:** Does this need to be both optional and nullable?
In what cases would we have `undefined` vs `null`?

Reply:
I'm not sure... I think I may have added both because Typescript was yelling at me otherwise 😬  Will revisit this.